src/framework/theme/components/chat/chat.component.ts

```diff
@@ -111,6 +116,18 @@ import { NbChatMessageComponent } from './chat-message.component';
  *
  * @stacked-example(Chat Sizes, chat/chat-sizes.component)
  *
+ * Available to provide custom template for chat messages <br/>
```

Comment:
Here we need a detailed text explanation on how to use `ngCustomMessage` directive. Something like:
You can provide a template for you own message types via the `ngCustomMessage` directive. First, you need to provide a message template. To do this write a template of your message in the `nb-chat` element, mark its root element with `*ngCustomMessage` directive and define it's type as a value of `*ngCustomMessage="my-custom-type"`.
`// code example of a message template with directive`
`// Important note`
Than, you need to set `type` property of the message which should be rendered via you custom template to the value you passed to the `nbCustomMessage` directive.
`// code example of message object with a custom message type`
...
`@stacked-example(Custom message, chat/chat-custom-message.component)`